### PR TITLE
docs: improve minikube instructions in get started section

### PR DESCRIPTION
**Pull request by sheeeng**
### Proposed changes

Give assurance that folks can use local minikube for learning Pulumi.

### CODE REVIEW

File: content/docs/get-started/kubernetes/deploy-changes.md [hunk 1]
```diff
@@ -60,7 +60,17 @@ Duration: 12s
 
 You can see we now have an `ip` [stack output]({{< relref "/docs/intro/concepts/stack#outputs" >}}) that we can `curl` to get the output from the service.
 
-> **Note:** minikube does not support type LoadBalancer; if you are deploying to minikube, make sure to run kubectl port-forward svc/frontend 8080:80 to forward the cluster port to the local machine and access the service via localhost:8080.
+> **Using Minikube:** Note that `minikube` does not support type LoadBalancer. If you are deploying to `minikube`, see following example to run `kubectl port-forward service/YOUR_SERVICE_NAME 8080:80` command to forward the cluster port to the local machine. Then, we can access the service via `curl http://localhost:8080`, which will get the same result as `curl $(pulumi stack output ip)` as in the environment without using `minikube`.
```
Comment (justinvp) [inline content/docs/get-started/kubernetes/deploy-changes.md]:
```suggestion
> **Using Minikube:** Note that Minikube does not support type `LoadBalancer`. If you are deploying to Minikube, see the following example to run the `kubectl port-forward service/YOUR_SERVICE_NAME 8080:80` command to forward the cluster port to the local machine. Then, the service can be accessed via `curl http://localhost:8080`, which will get the same result as `curl $(pulumi stack output ip)` as in the environment without using Minikube.
```

File: content/docs/get-started/kubernetes/modify-program.md [hunk 2]
```diff
@@ -243,7 +243,9 @@ class Program
 
 Our program now creates a service to access the NGINX deployment, and requires a new [config]({{< relref "/docs/intro/concepts/config" >}}) value to indicate whether the program is being deployed to Minikube or not.
 
-The required config value must be set for the stack using `pulumi config set isMinikube <true|false>`:
+The configuration value can be set for the stack using `pulumi config set isMinikube <true|false>` command.
+
+If you are currently using `minikube`, set `isMinikube` to `true`. Else, set `isMinikube` to `false` as shown in the following command.
```
[the remark this answers is not in the document]
Comment (sheeeng) [reply content/docs/get-started/kubernetes/modify-program.md]: Thanks you for the review. I have updated these suggestions with a newer commit.

File: content/docs/get-started/kubernetes/deploy-changes.md [hunk 3]
```diff
@@ -60,7 +60,17 @@ Duration: 12s
...
+
```
[the remark this answers is not in the document]
Comment (sheeeng) [reply content/docs/get-started/kubernetes/deploy-changes.md]: Thanks you for the review. I have updated these suggestions with a newer commit.

**Comment by sheeeng** [comment]
Thanks you for the detailed review. I really appreciate them.
These suggestions are included in a new commit.
Please let me know if you have more suggestions for this pull request.

File: content/docs/get-started/kubernetes/deploy-changes.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (sheeeng) [reply content/docs/get-started/kubernetes/deploy-changes.md]: These suggestions violated lint rules.

```text
/home/travis/build/pulumi/docs/content/docs/get-started/kubernetes/deploy-changes.md:

Line 64: Fenced code blocks should be surrounded by blank lines.
```

File: content/docs/get-started/kubernetes/deploy-changes.md [hunk 4]
```diff
@@ -60,7 +60,19 @@ Duration: 12s
...
+> NAME             TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)   AGE
+> kubernetes       ClusterIP   10.96.0.1       <none>        443/TCP   26h
+> nginx-3hq3kux6   ClusterIP   10.96.185.206   <none>        80/TCP    15m
+> ```
+>
+> ```bash
+> $ kubectl port-forward service/nginx-3hq3kux6 8080:80
+> Forwarding from 127.0.0.1:8080 -> 80
+> Forwarding from [::1]:8080 -> 80
+> ```
```
Comment (justinvp) [inline content/docs/get-started/kubernetes/deploy-changes.md]: Sorry, I should have been more clear: The `>` should only be in front of any empty lines and the lines that start with the three backticks and `bash` to start the code fense.

If `>` is in front of the lines _inside_ the code fence and the line of the closing three backticks, it won't render the way we want:

Screen Shot 2020-01-23 at 12 07 54 PM

The fix is to write it like this:

~~~markdown
> **Using Minikube:** Note that Minikube does not support type `LoadBalancer`. If you are deploying to Minikube, see the following example to run the `kubectl port-forward service/YOUR_SERVICE_NAME 8080:80` command to forward the cluster port to the local machine. Then, the service can be accessed via `curl http://localhost:8080`, which will get the same result as `curl $(pulumi stack output ip)` as in the environment without using Minikube.
>
> ```bash
$ kubectl get service
NAME             TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)   AGE
kubernetes       ClusterIP   10.96.0.1       <none>        443/TCP   26h
nginx-3hq3kux6   ClusterIP   10.96.185.206   <none>        80/TCP    15m
```
>
> ```bash
$ kubectl port-forward service/nginx-3hq3kux6 8080:80
Forwarding from 127.0.0.1:8080 -> 80
Forwarding from [::1]:8080 -> 80
```
~~~

Which renders the way we want.:

Screen Shot 2020-01-23 at 12 11 06 PM

File: content/docs/get-started/kubernetes/deploy-changes.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (sheeeng) [reply content/docs/get-started/kubernetes/deploy-changes.md]: I have moved the updated paragraph to the end of the document.

[the remark this answers is not in the document]
Comment (justinvp) [reply content/docs/get-started/kubernetes/deploy-changes.md]: @zchase, is there an easy way to relax the linting rules here so we can have code blocks inside the blockquote?

File: content/docs/get-started/kubernetes/deploy-changes.md (hunk 4 above)
Comment (sheeeng) [reply content/docs/get-started/kubernetes/deploy-changes.md]: Thank you for the further clarification.

Is it okay if we move the updated paragraph to the bottom of the file?

File: content/docs/get-started/kubernetes/deploy-changes.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (zchase) [reply content/docs/get-started/kubernetes/deploy-changes.md]: @justinvp wrapping the code in a disable comment for the linter should disable everything:
```
<!-- markdownlint-disable blanks-around-fences -->
...
<!-- markdownlint-enable blanks-around-fences -->
```

File: content/docs/get-started/kubernetes/deploy-changes.md (hunk 4 above)
Comment (justinvp) [reply content/docs/get-started/kubernetes/deploy-changes.md]: I'd prefer to keep it in a blockquote, but I am fine with it either in the original spot or at the bottom.

So (disabling the lint rules):

~~~markdown
<!-- markdownlint-disable blanks-around-fences -->
> **Using Minikube:** Note that Minikube does not support type `LoadBalancer`. If you are deploying to Minikube, see the following example to run the `kubectl port-forward service/YOUR_SERVICE_NAME 8080:80` command to forward the cluster port to the local machine. Then, the service can be accessed via `curl http://localhost:8080`, which will get the same result as `curl $(pulumi stack output ip)` as in the environment without using Minikube.
>
> ```bash
$ kubectl get service
NAME             TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)   AGE
kubernetes       ClusterIP   10.96.0.1       <none>        443/TCP   26h
nginx-3hq3kux6   ClusterIP   10.96.185.206   <none>        80/TCP    15m
```
>
> ```bash
$ kubectl port-forward service/nginx-3hq3kux6 8080:80
Forwarding from 127.0.0.1:8080 -> 80
Forwarding from [::1]:8080 -> 80
```
<!-- markdownlint-enable blanks-around-fences -->
~~~

Comment (sheeeng) [reply content/docs/get-started/kubernetes/deploy-changes.md]: Thank you for the feedback. I have updated the pull request.

File: content/docs/get-started/kubernetes/deploy-changes.md [hunk 5]
```diff
@@ -60,7 +60,22 @@ Duration: 12s
...
```
Comment (justinvp) [inline content/docs/get-started/kubernetes/deploy-changes.md]: Still need to remove the `>` _inside_ the code blocks. After that, it should be ready to merge! Thanks, again for the improvements here!

~~~suggestion
$ kubectl get service
NAME             TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)   AGE
kubernetes       ClusterIP   10.96.0.1       <none>        443/TCP   26h
nginx-3hq3kux6   ClusterIP   10.96.185.206   <none>        80/TCP    15m
```
>
> ```bash
$ kubectl port-forward service/nginx-3hq3kux6 8080:80
Forwarding from 127.0.0.1:8080 -> 80
Forwarding from [::1]:8080 -> 80
```
~~~

Comment (sheeeng) [reply content/docs/get-started/kubernetes/deploy-changes.md]: Hmm. My local Visual Studio Code does not properly render under Markdown preview without these block quote in the code block.

I will change according to your feedback. Pull request updated.

Comment (justinvp) [reply content/docs/get-started/kubernetes/deploy-changes.md]: I pulled down the changes in your PR and confirmed that Hugo renders it as expected with these changes. Thanks!.

Comment (sheeeng) [reply content/docs/get-started/kubernetes/deploy-changes.md]: Great! Good to hear. I did not know that the website is using Hugo. 😅

**Pull request merged**